<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetStrainVideo</class>
 <widget class="QWidget" name="WidgetStrainVideo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="WidgetResult" name="widgetResult"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btnPrevBeat">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/icons/icons/resultset_first.png</normaloff>:/icons/icons/resultset_first.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnPrev">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/icons/icons/resultset_previous.png</normaloff>:/icons/icons/resultset_previous.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblIndex">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="horizontalSlider">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnNext">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/icons/icons/resultset_next.png</normaloff>:/icons/icons/resultset_next.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnNextBeat">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/icons/icons/resultset_last.png</normaloff>:/icons/icons/resultset_last.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>WidgetResult</class>
   <extends>QGraphicsView</extends>
   <header>widgetresult.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>horizontalSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>WidgetStrainVideo</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>174</x>
     <y>467</y>
    </hint>
    <hint type="destinationlabel">
     <x>5</x>
     <y>398</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>display(int)</slot>
 </slots>
</ui>
